apps/prairielearn/src/pages/instructorAssessmentInstance/instructorAssessmentInstance.ejs

```diff
@@ -40,6 +40,53 @@
             <tr><th>Instance</th><td colspan="2"><%= assessment_instance.number %>
                 (<a href="<%= plainUrlPrefix %>/course_instance/<%= course_instance.id %>/assessment_instance/<%= assessment_instance.id %>">student view</a>)
             </td></tr>
+            <% if (instance_user) { %>
+            <tr>
+              <th>Fingerprint Changes</th>
```

Comment:
Let's also add a section to the docs (I'm not sure where, maybe under Assessments) about fingerprint tracking and changes.

Reply:
I have now added a section under _Assessments_ in the in docs to explain client fingerprints are and what changes could be an indication of.